src/stores/saved-cities.store.ts

```diff
@@ -14,6 +14,10 @@ export const useSavedCitiesStore = defineStore("saved-cities", {
     },
     actions: {
         addCity(city: City) {
+            if (city.name === "Your current location") {
```

Comment:
Would this update the "your current location" lat/lng if your are at another location?

Reply:
yes, the old one gets removed and replaced